src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum = 0;
+  const dayRent = 40;
+  const littleDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    sum = dayRent * days;
+  } else if (days < 7) {
+    sum = dayRent * days - littleDiscount;
+  } else {
+    sum = dayRent * days - bigDiscount;
+  }
```

Comment:
Don't repeat yourself: create variable for dayRent * days value
Check point 3 in checklist